pkg/credentials/env_minio.go

```diff
@@ -22,10 +22,12 @@ import "os"
 // A EnvMinio retrieves credentials from the environment variables of the
 // running process. EnvMinioironment credentials never expire.
 //
-// EnvMinioironment variables used:
+// Environment variables used:
 //
 // * Access Key ID:     MINIO_ACCESS_KEY.
```

Comment:
Shouldn't we remove the deprecated env vars ?

Reply:
No not everyone is expected to migrate, and we shall never remove these ENVs from MinIO either @vadmeste